driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/syncadapter/SyncMongoCollection.java

```diff
@@ -743,22 +743,30 @@ public void drop(final ClientSession clientSession) {
 
     @Override
     public String createIndex(final Bson keys) {
-        throw new UnsupportedOperationException();
+        SingleResultSubscriber<String> subscriber = new SingleResultSubscriber<>();
```

Comment:
The new JSON tests contain commands for `createIndex` and `dropIndex` that need to be supported in reactive streams. I have added support for these commands here.